packages/cas/src/tests/testSuite.ts

```diff
@@ -124,6 +127,8 @@ const testSuite = (cas: ICasService): void => {
         it(`should write and read a ${key} file`, async () => {
           const { cid, content, jsonStr } = ionVectors[key];
           const compressedBuffer = await gzipAsync(jsonStr);
+          // https://docs.fileformat.com/compression/gz/#operating-system
```

Comment:
ouch... are we sure this works with ION?

Reply:
I don't think that should be effected by this change.  This makes sure the test gzip's the same buffer contents as the fixture, there is a byte that is different depending on the os the test is running on, since I was running the test on a mac I had to modify this byte to match unix fixture.